cmake/Modules/GrPython.cmake

```diff
@@ -110,6 +110,18 @@ if os.name == 'nt':
 endif()
 file(TO_CMAKE_PATH ${GR_PYTHON_DIR} GR_PYTHON_DIR)
 
+########################################################################
+# Sets the python relative installation directory GR_PYTHON_RELATIVE
+########################################################################
+if(NOT DEFINED GR_PYTHON_RELATIVE)
+execute_process(
+  COMMAND "${PYTHON_EXECUTABLE}" -c "if True:
```

Comment:
why the "if True:"? that command is always True, of course ...so seems unnecessary!

Reply:
I was attempting to perform basic error checking.  CMake scripting is not really in my area.  I will remove this and test on Monday.